docs/business-apps/power-apps/get-started/create-your-first-custom-form.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Customize a form for a SharePoint list
+description: Customize a form for a SharePoint list
+ms.date: 06/03/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+# Customize a form for a SharePoint list
+
+Using Power Apps, you can easily customize a form for a SharePoint list that works best for your team or organization.
+
+In this tutorial, you will create a custom form with a user profile photo, a read-only field, and a conditionally-visible field.
+
+## Required setup
+
+Before starting, make sure your SharePoint site is set up with the [required lists and libraries](https://docs.microsoft.com/sharepoint/dev/business-apps/get-started/set-up-sharepoint-site-lists-libraries).
+
+## Create a custom form
+
+1. In your SharePoint site, browse to the **Projects** list.
+
+2. To create and manage flows for a list or a library, from the command bar, on the list or library page, select **Power Apps**. More options appear:
```

Comment:
Don't use numbers in bullets... all numbered bullets should start with `1.`. Let the rendering engine add the actual numbers. This simplifies maintenance going forward.

Reply:
I'm confused. What is the requested change here?

Reply:
The bullets below (create an app, etc.) aren't using numbers... and the instruction steps are using numbers. Does one or both of them need to be changed?

Reply:
@steven-jia lines 21 & 23 in the file are markdown bullets... one is "1." and the other is "2.". When doing markdown bullets, don't use explicit numbers... every single bullet should be "1." and you should rely on the MSDocs markdown rendering engine to render the correct numbers. This simplifies maintenance going forward.

You can preview this in VSCode... if the numbers aren't rendering the correct sequence or are restarting, there's an issue with your indentation that should be resolved.

Look at this [file for a good example](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/spfx/web-parts/get-started/connect-to-sharepoint.md). When you see the rendered bullets, they are a sequence of numbers (1-2-3-4...). However, if you look at the raw source, all you see is (1. 1. 1. 1.).

Reply:
Thanks for the clarification! I've made the changes to numbering.